ethexe/runtime/common/src/transitions.rs

```diff
@@ -28,6 +28,8 @@ use ethexe_common::{
 };
 use gprimitives::{ActorId, H256};
 
+pub type Expiry = u32;
```

Comment:
more likely `type BlockNumber`, since expiry points at specific height of the chain